.bit/responses/1.7-Week 1 Step 7.md

```diff
@@ -93,4 +81,16 @@ context.res = {
 }
 ```
 <br><br/>
-</details>
\ No newline at end of file
+</details>
+
+### 🚧 Test your Work
```

Comment:
We need to keep the Test your Work sections at the top of the issue in case more advanced students don't need the instructions. That way, they don't have to scroll to the very bottom every time.